src/test/java/mate/academy/spring/boot/controllerTests/BookControllerTest.java

```diff
@@ -0,0 +1,308 @@
+package mate.academy.spring.boot.controllerTests;
+
+import mate.academy.spring.boot.dto.book.BookDto;
+import mate.academy.spring.boot.dto.book.BookSearchParameters;
+import mate.academy.spring.boot.dto.book.CreateBookRequestDto;
+import mate.academy.spring.boot.dto.book.UpdateBookRequestDto;
+import mate.academy.spring.boot.service.impl.BookServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.testcontainers.shaded.org.apache.commons.lang3.builder.EqualsBuilder;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers.springSecurity;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class BookControllerTest {
+    @LocalServerPort
+    private int port;
+    @Autowired
+    private WebApplicationContext context;
+    @Autowired
+    private ObjectMapper objectMapper;
+    @Autowired
+    private BookServiceImpl bookService;
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    void beforeAll() {
+        mockMvc = MockMvcBuilders
+                .webAppContextSetup(context)
+                .apply(springSecurity())
+                .build();
+    }
+
+    @Test
+    @WithMockUser
+    @DisplayName("Should return List BookDtoS")
+    @Sql(scripts = "classpath:database/add-books-to-books-table.sql"
+            , executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/remove-books-from-books-table.sql"
+            , executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
```

Comment:
```suggestion
    @Sql(scripts = "classpath:database/add-books-to-books-table.sql",
              executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
    @Sql(scripts = "classpath:database/remove-books-from-books-table.sql",
              executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
```
Let’s  fix it everywhere